lib/hyacinth/digital_object/resource_attributes.rb

```diff
@@ -8,11 +8,11 @@ def self.included(base)
       end
 
       def resource_attributes
-        self.class.resource_attributes
+        self.class.send(__method__) # class method name same as instance method
       end
 
       def resource_import_attributes
-        self.class.resource_attributes
```

Comment:
this is now delegating to a different class method (before both methods delegated to the `resource_attributes` class method), right?

Reply:
That's correct.  That was actually a mistake before.  The instance method should behave the same way as its same-name class method.